cypress/e2e/timeline/timeline.spec.ts

```diff
@@ -776,11 +788,13 @@ describe("Timeline", () => {
             cy.getComposer().type(`${MESSAGE}{enter}`);
 
             // Reply to the message
-            cy.contains(".mx_RoomView_body .mx_EventTile_line", "Hello world")
-                .realHover()
+            cy.get(".mx_EventTile_last")
                 .within(() => {
-                    cy.findByRole("button", { name: "Reply" }).click();
-                });
+                    cy.findByText("Hello world", { timeout: 1000 });
```

Comment:
Can you remove the timeout? Sometimes tests run slowly. This is a possible cause for flaky tests.